WordPressAuthenticatorTests/Authenticator/WordPressSourceTagTests.swift

```diff
@@ -0,0 +1,175 @@
+import XCTest
+import WordPressAuthenticator
+
+class WordPressSourceTagTests: XCTestCase {
+    
+    func testGeneralLoginSourceTag() {
+        let tag = WordPressSupportSourceTag.generalLogin
+        
+        let nameExpectation = "generalLogin"
+        let originExpectation = "origin:login-screen"
+        
+        XCTAssertEqual(tag.name, nameExpectation)
+        XCTAssertEqual(tag.origin, originExpectation)
```

Comment:
You could keep the tests more compact by inlining the expected values:

```suggestion
        XCTAssertEqual(tag.name, "generalLogin")
        XCTAssertEqual(tag.origin, "origin:login-screen")
```